README.md

```diff
@@ -801,6 +802,16 @@ These options should contain something like:
 - `PIWIK_URL=piwik.example.org`
 - `PIWIK_SITE_ID=42`
 
+### Exposing ssh port in dockerized gitlab-ce
```

Comment:
I think we should have a separate document for this and linking only here to document.

So than we could have a documented in the docs. 

As an example have a look on https://github.com/sameersbn/docker-gitlab/blob/master/docs/container_registry.md or https://github.com/sameersbn/docker-gitlab/blob/master/docs/s3_compatible_storage.md